src/types/chat.rs

```diff
@@ -232,6 +241,174 @@ impl Chat {
     }
 }
 
+/// Getters
```

Comment:
Does it appear in the docs?

Reply:
Yes

![image](https://user-images.githubusercontent.com/38225716/117028009-f2824f00-ad05-11eb-910c-79c29a02345b.png)